src/bin/nydusd/main.rs

```diff
@@ -363,6 +373,80 @@ fn handle_rlimit_nofile_option(args: &ArgMatches, option_name: &str) -> Result<(
     Ok(())
 }
 
+fn send_unix_socket_request(socket_path: &str, method: &str, url: &str) -> Option<String> {
+    let mut stream = match UnixStream::connect(socket_path).ok() {
+        Some(stream) => stream,
+        None => return None,
+    };
+
+    let request = format!("{} {} HTTP/1.1\r\nHost: localhost\r\n\r\n", method, url);
+    if stream.write_all(request.as_bytes()).is_err() {
```

Comment:
Show sth. in log or returned value may be helpful to address problem

Reply:
> Show sth. in log or returned value may be helpful to address problem

I will do it.